liquibase-couchbase/src/test/java/system/change/CreateScopeSystemTest.java

```diff
@@ -0,0 +1,30 @@
+package system.change;
+
+import com.couchbase.client.java.Bucket;
+import liquibase.Liquibase;
+import liquibase.ext.couchbase.operator.BucketOperator;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import system.LiquibaseSystemTest;
+
+import static common.constants.ChangeLogSampleFilePaths.CREATE_SCOPE_TEST;
+import static common.constants.TestConstants.TEST_BUCKET;
+import static common.matchers.CouchbaseBucketAssert.assertThat;
+
+public class CreateScopeSystemTest extends LiquibaseSystemTest {
+
+    private static final String SCOPE_NAME = "scopeToRollback";
+    private static final Bucket BUCKET = cluster.bucket(TEST_BUCKET);
```

Comment:
`bucketOperator.getBucket()`could be used for the same idea (if only you don't want to re-create this bucket)

Reply:
++